resources/META-INF/plugin.xml

```diff
@@ -28,6 +28,7 @@
   <depends>com.intellij.modules.xdebugger</depends>
   <depends>com.intellij.modules.coverage</depends>
   <depends>org.jetbrains.plugins.yaml</depends>
+  <depends>org.jetbrains.android</depends>
```

Comment:
I think this change means we will not be able to launch Android Studio in the debugger. But I haven't been able to do that since late last year, due to a Kotlin compiler bug, so that's not a blocker. Anyway, if the bug gets fixed I can add some code to the `plugin` tool to fix up the `plugin.xml` content.

Reply:
Great. It is just that I wasn’t able to fix the class-not-found with this change